demo-pages-shared-styles.js

```diff
@@ -1,3 +1,14 @@
+/**
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at
+http://polymer.github.io/LICENSE.txt The complete set of authors may be found at
+http://polymer.github.io/AUTHORS.txt The complete set of contributors may be
+found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by Google as
+part of the polymer project is also subject to an additional IP rights grant
+found at http://polymer.github.io/PATENTS.txt
+*/
+
 import '@polymer/polymer/polymer-legacy.js';
 import '@polymer/iron-flex-layout/iron-flex-layout.js';
 import '@polymer/font-roboto/roboto.js';
```

Comment:
@e111077 should the code on the next line (that I can't comment on because github) be replaced with the new  template string syntax we use in demos? i.e.
```js
html`<dom-module>...</dom-module>`
```

Reply:
It would be preferable though does the same thing /shrug

Reply:
yeah, i agree. changing!